tetris.js

```diff
@@ -308,5 +299,21 @@ module.exports = {
 		let coordinateDifference_X = coordinatesPivot.y - coordinatesElement.y
 		let coordinateDifference_Y = coordinatesPivot.x - coordinatesElement.x
 		return { x: coordinatesPivot.x - coordinateDifference_X, y: coordinatesPivot.y + coordinateDifference_Y }
+	},
+	rotatesArrayOfCoordinates: function(arrayCoordinates, coordinatePivot){
+		let localArrayCoordinates = []
+		let localCoordinatePivot = coordinatePivot[0]
+		for(let arrayElement of arrayCoordinates){
+			localArrayCoordinates.push(this.rotateElement(arrayElement,localCoordinatePivot))	
+		}
+		return localArrayCoordinates.sort(function(firstCoordinate,secondCoordinate){
+			let coordinates_X = firstCoordinate.x - secondCoordinate.x;
+			if(firstCoordinate.x === secondCoordinate.x){
+				let coordinates_Y = firstCoordinate.y - secondCoordinate.y;
+				return coordinates_X, coordinates_Y;
```

Comment:
Again, how does this work? You should always return number

Reply:
it checks the elements of the array and sorts them

Reply:
Ok, if you are so confident, then create separate PR with sorting function and tests for it